intermediate_source/torchvision_tutorial.py

```diff
@@ -35,10 +35,10 @@
 # The reference scripts for training object detection, instance
 # segmentation and person keypoint detection allows for easily supporting
 # adding new custom datasets. The dataset should inherit from the standard
-# ``torch.utils.data.Dataset`` class, and implement ``__len__`` and
-# ``__getitem__``.
+# ``torch.utils.data.Dataset`` class, and implement ``_len__`` and
+# ``_getitem__``.
```

Comment:
Just FYI, the search and replace seems to have cause false positives like here.

(Also, ending rst links with 2 backticks `__` instead of 1 may help avoiding issues when links are duplicated... not sure it will be a problem here, but something to consider)

Reply:
Just fixed it